### DISCUSSION TITLE
How to Contribute Financially??

### DISCUSSION THREAD

**Discussion by gbeyer3** [AlexandreRouma/SDRPlusPlus#863] [General]
Alxandre, how can we appreciative users of SDR++ contribute to your efforts financially?  I see nothing in the README.  You and others have put a LOT of work into this -- I cannot contribute with development, but I can send you cup(s) of coffee :-)   Best regards.  Greg

**Comment by AlexandreRouma** [comment]
Currently the only way is [Patreon](https://www.patreon.com/ryzerth), which I know isn't ideal for everyone.
I completely got rid of paypal due to having loads of issue with them in the past few months.
